modules/apps/frontend-icons/frontend-icons-web/src/main/java/com/liferay/frontend/icons/web/internal/category/FrontendIconsConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.icons.web.internal.category;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Bryce Osterhaus
+ */
+@ExtendedObjectClassDefinition(
+	category = "frontend-icons", generateUI = false,
+	scope = ExtendedObjectClassDefinition.Scope.COMPANY
+)
+@Meta.OCD(
+	id = "com.liferay.frontend.icons.web.internal.category.FrontendIconsConfiguration",
+	localization = "content/Language",
+	name = "frontend-icons-configuration-name"
+)
+public interface FrontendIconsConfiguration {
```

Comment:
What is the purpose of an empty configuration interface? :thinking:

Reply:
I believe this is needed for adding the configuration to the system setting 🤔  but I may be wrong

Reply:
Yes, that's what I think after seeing the whole thing in action :+1: .Thx.